src/schema/v2/__tests__/partner.test.js

```diff
@@ -462,10 +507,10 @@ describe("Partner type", () => {
     })
 
     it("returns shows", async () => {
-      const query = `
+      const query = gql`
```

Comment:
the next few changes can be ignored. just added the `gql` here for the formatting